services/core-web/src/components/admin/complianceCodes/ComplianceCodeViewEditForm.tsx

```diff
@@ -91,42 +91,39 @@ const ComplianceCodeViewEditForm: FC<{
         isEditMode={isEditMode}
         isModal={true}
       >
-        <Row>
-          <Steps current={currentStep}>
-            {steps.map((step) => (
-              <Steps.Step key={step.title} title={step.title} />
-            ))}
-          </Steps>
+        <Steps current={currentStep}>
+          {steps.map((step) => (
+            <Steps.Step key={step.title} title={step.title} />
+          ))}
+        </Steps>
+        {steps[currentStep].content}
```

Comment:
this looks like a big change but it's really just indentation from taking out a `<Row>` or two